lua/entities/gmod_wire_pod.lua

```diff
@@ -471,6 +472,12 @@ function ENT:TriggerInput(name, value)
 		if value:IsNPC() then return end
```

Comment:
Could you update this to use the TypeID and IsValid check, instead of IsPlayer IsNPC? All the rest of the code looks good.